ingest.py

```diff
@@ -14,23 +30,47 @@
 load_dotenv()
 
 
+# Map file extensions to document loaders and their arguments
+LOADER_MAPPING = {
+    ".csv": (CSVLoader, {}),
+    # ".docx": (Docx2txtLoader, {}),
```

Comment:
Should we remove this commented out line?

Reply:
I added this as a comment as a reminder that there is another type of reader for docx files.

Maybe one would like to choose one or the other in the future.